istioctl/cmd/istioctl/convert/cmd.go

```diff
@@ -115,8 +115,6 @@ func convertConfigs(readers []io.Reader, writer io.Writer) error {
 		model.HTTPAPISpecBinding,
 		model.QuotaSpec,
 		model.QuotaSpecBinding,
```

Comment:
Do you need to add `model.AuthenticationPolicy,` here/

Reply:
I'm not sure what is this for. From comment on the cobra.Command, looks like it for the tool to convert one CRD to another, which is not applicable for authn policy.